src/shared/components/person/person-listing.tsx

```diff
@@ -24,64 +24,47 @@ export class PersonListing extends Component<PersonListingProps, any> {
   }
 
   render() {
-    const person = this.props.person;
+    const { person, useApubName } = this.props;
     const local = person.local;
-    let apubName: string, link: string;
+    let link: string;
+    let serverStr: string | undefined = undefined;
+
+    const name = useApubName ? person.name : person.display_name ?? person.name;
 
     if (local) {
-      apubName = `@${person.name}`;
       link = `/u/${person.name}`;
     } else {
-      const domain = hostname(person.actor_id);
-      apubName = `@${person.name}@${domain}`;
+      serverStr = `@${hostname(person.actor_id)}`;
       link = !this.props.realLink
-        ? `/u/${person.name}@${domain}`
+        ? `/u/${person.name}${serverStr}`
         : person.actor_id;
     }
 
-    let displayName = this.props.useApubName
-      ? apubName
-      : person.display_name ?? apubName;
-
-    if (this.props.showApubName && !local && person.display_name) {
-      displayName = `${displayName} (${apubName})`;
-    }
-
+    const classes = classNames(
+      "person-listing d-inline-flex align-items-baseline",
+      {
+        "text-muted": this.props.muted,
+        "text-info": !this.props.muted,
+      },
+    );
```

Comment:
A ternary using `muted` would be more concise.

Reply:
This is using that special `classNames` function, it can't work in that way.